global/meta/models.yml

```diff
@@ -147,6 +147,10 @@ organization:
     to: mediafile/owner_id
     on_delete: CASCADE
     restriction_mode: A
+  user_ids:
+    type: relation-list
+    restriction_mode: A
```

Comment:
This is the place, where for performance reasons I would like you to use another restriction_mode.

Reply:
I chatted with @r-peschke about this topic. I will change the restriction_mode of organization/user_ids to C.

Reply:
Done.